semcore/popper/__tests__/index.test.jsx

```diff
@@ -302,4 +302,46 @@ describe('focus control', () => {
     expect(document.activeElement.nodeName).toBe('DIV');
     vi.useRealTimers();
   });
+
+  test('prevent focus return', () => {
```

Comment:
Basically I am very surprised that without `preventFocusCatch` functionality this test is passing 🤔

Reply:
removed test, we don't need it